routes/list.js

```diff
@@ -1,9 +1,28 @@
 'use strict';
 
+const fs = require('fs');
+const _ = require('lodash');
+const Handlebars = require('handlebars');
+
 exports.list = {
   path: '/',
   method: 'GET',
   handler(request, reply) {
-    reply('ok');
+    const templateDir = request.server.settings.app.views.path;
+    fs.readdir(templateDir, (err, files) => {
+      if (err) {
+        return reply(err);
+      }
+
+      fs.readFile('files/index.html', 'utf8', (fileErr, contents) => {
```

Comment:
Since this is micro mail - the views folder is really for the templates which is a separate thing in our case. This for now is just a "One Off" template. @jgallen23